src/applications/vaos/appointment-list/pages/RequestedAppointmentsPage/RequestedAppointmentsPage.unit.spec.jsx

```diff
@@ -66,14 +67,10 @@ describe('VAOS Component: RequestedAppointmentsPage', () => {
       },
       requestedPeriods: [
         {
-          start: moment(startDate)
-            .add(3, 'days')
-            .format('YYYY-MM-DDTHH:mm:ss[Z]'),
+          start: format(addDays(startDate, 3), "yyyy-MM-dd'T'HH:mm:ssXXX"),
```

Comment:
Hmmm why are we starting to include the timezone information via `XXX` now?

Reply:
On second though I guess it makes sense to include it instead of using the local time as UTC. Just want to make sure that adding a timezone info is allowed on the MAP endpoint?